cider/tests/utils/CiderBatchChecker.cpp

```diff
@@ -144,6 +144,47 @@ bool CiderBatchChecker::checkOneScalarBatchEqual(const ScalarBatch<T>* expected_
   return true;
 }
 
+template <>
+bool CiderBatchChecker::checkOneScalarBatchEqual<bool>(
```

Comment:
Not quite sure about the implementation choice here. The check is based on bitwise ops instead of `memcmp`. My intention is to use validity bitmap as a mask to avoid failure cases caused by different padding or null values. But perhaps it can be done in a more efficient way, or maybe I should have just used `memcmp`.

Reply:
Maybe you can use memcmp but with only compare the correct length of bits.